scripts/secnetperf.ps1

```diff
@@ -0,0 +1,300 @@
+    } -AsJob
+} else {
+    $Job = Invoke-Command -Session $Session -ScriptBlock {
+        $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:/home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/"
+        chmod +x /home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/secnetperf
+        /home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/secnetperf -exec:maxtput
+    } -AsJob
+}
+
+
+# Wait for the server to start.
+Write-Output "Waiting for server to start..."
+Start-Sleep -Seconds 10
+
+# Run secnetperf on the client.
+Write-Output "Running tests on the client..."
+
+####################################################################################################
+
+    # TEST EXECUTION
+
+####################################################################################################
+
+# TODO:
+
+$SQL = @"
+
+INSERT OR IGNORE INTO Secnetperf_builds (Secnetperf_Commit, Build_date_time, TLS_enabled, Advanced_build_config)
+VALUES ('$MsQuicCommit', '$(Get-Date -Format "yyyy-MM-dd HH:mm:ss")', 1, 'TODO');
+
+INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Secnetperf_build_ID, Kernel_mode, Run_arguments, Test_name)
+VALUES ('throughput-upload-quic-$MsQuicCommit', '$MsQuicCommit', 0, '-target:netperf-peer -exec:maxtput -upload:10s', 'throughput-upload-quic');
+
+INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Secnetperf_build_ID, Kernel_mode, Run_arguments, Test_name)
+VALUES ('throughput-upload-tcp-$MsQuicCommit', '$MsQuicCommit', 0, '-target:netperf-peer -exec:maxtput -upload:10s -tcp:1', 'throughput-upload-tcp');
+
+INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Secnetperf_build_ID, Kernel_mode, Run_arguments, Test_name)
+VALUES ('throughput-download-quic-$MsQuicCommit', '$MsQuicCommit', 0, '-target:netperf-peer -exec:maxtput -download:10s', 'throughput-download-quic');
+
+INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Secnetperf_build_ID, Kernel_mode, Run_arguments, Test_name)
+VALUES ('throughput-download-tcp-$MsQuicCommit', '$MsQuicCommit', 0, '-target:netperf-peer -exec:maxtput -download:10s -tcp:1', 'throughput-download-tcp');
+
+"@
+
+$exe = ".\artifacts\bin\windows\x64_Release_schannel\secnetperf.exe"
+
+if (!$isWindows) {
+    $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:./artifacts/bin/linux/x64_Release_openssl/"
+    $exe = "./artifacts/bin/linux/x64_Release_openssl/secnetperf"
+    chmod +x ./artifacts/bin/linux/x64_Release_openssl/secnetperf
+}
+
+$json = @{}
+
+# TODO: Make a more elaborate execution strategy instead of just a list of commands. Also add more tests.
+
+$testIds = @(
+    "throughput-upload-quic-$MsQuicCommit",
+    "throughput-upload-tcp-$MsQuicCommit",
+    "throughput-download-quic-$MsQuicCommit",
+    "throughput-download-tcp-$MsQuicCommit",
+    "rps-1conn-1stream-quic-$MsQuicCommit",
+    "rps-1conn-1stream-tcp-$MsQuicCommit",
+    "hps-quic-$MsQuicCommit",
+    "hps-tcp-$MsQuicCommit"
+)
+
+$commands = @(
```

Comment:
I'd like to remove the `-tcp:` option having to be explicitly specified in the list of commands. Every command can be run in QUIC or TCP mode. Let's remove it from the list below and dynamically add it in the loop below.